ubuntu/gpu/cuda-11.8/venv/Dockerfile

```diff
@@ -0,0 +1,149 @@
+
+FROM databricksruntime/gpu-base:cuda11.8
+
+WORKDIR /databricks
+
+# Install python 3.10 from ubuntu.
+# Install pip via get-pip.py bootstrap script and install versions that match Anaconda distribution.
+RUN apt-get update \
+  && apt-get install curl software-properties-common -y python3.10 python3.10-dev python3.10-distutils \
+  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
+  && /usr/bin/python3.10 get-pip.py pip==22.2.2 setuptools==58.0.4 wheel==0.37.0 \
+  && rm get-pip.py
+
+
+# virtualenv 20.0.24 introduced a periodic update feature, which attempts to update all
+# seeder packages every 14 days. This launches background processes that may interfere
+# with user cleanup and may allow users to inadvertently update pip to newer versions
+# incompatible with Databricks. Instead, we patch virtualenv to disable periodic updates per
+# https://virtualenv.pypa.io/en/latest/user_guide.html#embed-wheels-for-distributions. See
+# ML-13720 for more details.
+RUN /usr/local/bin/pip3.10 install --no-cache-dir virtualenv==20.16.3 \
```

Comment:
Should we use 20.16.7 to be consistent with DBR 14.0? https://github.com/databricks/universe/blob/master/spark/images/base/virtualenv-python/14.x/dbr-virtualenv-python.dockerfile.py#L22